frontend/pages/_document.tsx

```diff
@@ -0,0 +1,30 @@
+import Document from "next/document";
+import { ServerStyleSheet } from "styled-components";
+
+export default class MyDocument extends Document {
+  static async getInitialProps(ctx) {
+    const sheet = new ServerStyleSheet();
+    const originalRenderPage = ctx.renderPage;
+
+    try {
```

Comment:
We've got the `try {` and if this fails, we immediately call sheet.seal(); (in the finally) Idk how styled-comps work, but what's the point in 'sealing' something that may have potentially failed? You should have a fallback catch here if the ctx.renderPage() can fail.

Reply:
It's straight from the next js docs.

Reply:
Oh ok, I think it may be a little fuck up in their docs lol.

Reply:
may b lol.